### DISCUSSION TITLE
livecheck leaves artefacts

### DISCUSSION THREAD

**Discussion by ChristianKraemer** [Homebrew/discussions#5836] [Everyday usage] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.4.14-4-gf756ea4-dirty
ORIGIN: https://github.com/Homebrew/brew
HEAD: f756ea471b4e9eefcf8769becdb5bfd82795683d
Last commit: 39 minutes ago
Branch: master
Core tap HEAD: f14ce850cea11cda0915ca0d3fd111e007c9e3da
Core tap last commit: 16 minutes ago
Core cask tap HEAD: 228ace87cd20e37991daf2261d6029be4418908a
Core cask tap last commit: 38 minutes ago
Core cask tap JSON: 15 May 06:07 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.Y4Mrae0G1z/org.xquartz:0
HOMEBREW_MAKE_JOBS: 8
HOMEBREW_NO_ENV_HINTS: set
HOMEBREW_NO_INSTALL_FROM_API: set
HOMEBREW_SORBET_RUNTIME: set
Homebrew Ruby: 3.3.6 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/3.3.6/bin/ruby
CPU: octa-core 64-bit kabylake
Clang: 16.0.0 build 1600
Git: 2.47.1 => /usr/local/bin/git
Curl: 8.7.1 => /usr/bin/curl
macOS: 15.2-x86_64
CLT: 16.2.0.0.1.1733547573
Xcode: N/A
```

### Output of `brew doctor`

```shell
will be supplied in description
```

### Description of issue

What I experience is:

`xyz@universe Homebrew % brew doctor
Your system is ready to brew.
xyz@universe Homebrew % brew lc --newer-only --installed --quiet --full-name
Removing base64 (0.2.0)
xyz@universe Homebrew % brew doctor                                         
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: You have uncommitted modifications to Homebrew/brew.
If this is a surprise to you, then you should stash these modifications.
Stashing returns Homebrew to a pristine state but can be undone
should you later need to do so for some reason.
  cd /usr/local/Homebrew && git stash -u && git clean -d -f

Uncommitted files:
   D Library/Homebrew/vendor/bundle/ruby/3.3.0/gems/base64-0.2.0/LICENSE.txt
   D Library/Homebrew/vendor/bundle/ruby/3.3.0/gems/base64-0.2.0/lib/base64.rb`

I did this stash & clean thing a few times, but the error appears after every 'brew lc'.
How can I get rid of these files finally?

**Comment by p-linnane** [comment]
Try running [`brew update-reset`](https://docs.brew.sh/Manpage#update-reset-repository-).

**Comment by ChristianKraemer** [comment]
The 'update-reset' itself runs fine, but does not cure the situation. The 'lc' again removes some 'base64' stuff and leaves two files.

**Comment by daeho-ro** [comment]
Have you pushed to your fork? or origin? I think it should be a fork one.

**Comment by ChristianKraemer** [comment]
Did this in the past within a 'bump-cask-pr', but latest occurence weeks ago. Did nothing manually.

**Comment by ChristianKraemer** [comment, marked as answer]
Something happened in the background. When I just minutes ago did this 'lc & stash' cycle I got the output:

`Your system is ready to brew.
Fetching gem metadata from https://rubygems.org/.......
Fetching base64 0.2.0
Fetching sorbet-runtime 0.5.11709
Installing base64 0.2.0
Installing sorbet-runtime 0.5.11709
Bundle complete! 39 Gemfile dependencies, 30 gems now installed.
Bundled gems are installed into `./Library/Homebrew/vendor/bundle``

And the problems seems to be gone.
